src/App.jsx

```diff
@@ -27,6 +28,7 @@ function App() {
           <Route path="/login" element={<Login />} />
           <Route path="/logout" element={<Logout />} />
           <Route path="/verify-2fa" element={<Verify2FA />} />
+          <Route path="/register-staff" element={<RegisterStaff />} />
```

Comment:
Shouldn't this route be inside AppLayout?

Reply:
Yeah you're right, I have added the form into the AppLayout in my latest commit, also I have added a min width because the form looked too skinny

Reply:
Also, it needs to be inside /app to avoid re-rendering the sidebar

Reply:
Yes, I have tested it on mobile

Reply:
Resolved in https://github.com/FIS2425/frontend/pull/30/commits/680fb1e0b81b3c26429f410dc4d349848b139dbb